main.go

```diff
@@ -1,7 +1,58 @@
-package main
+package testing101
+import (
+	"context"
+	"time"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"go.mongodb.org/mongo-driver/mongo"
+	"github.com/labstack/echo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
 
-import "fmt"
-
-func main() {
-	fmt.Println("18.20 at Sep 07 2016 7:20 IR721")
+type test struct {
+ 	Lat string `json:"lat"`
+	Lng string `json:"lng"`
+}
+func JsonHandler(c echo.Context) (err error) {
+  t :=test{}
+  defer c.Request().Body.Close()
+  b,err := ioutil.ReadAll(c.Request().Body)
+  if err!=nil {
+	  log.Printf("failed to rendering request body:%s",err)
+	  return c.String(http.StatusInternalServerError,"")
+  }
+  err=json.Unmarshal(b,&t)
+  if err!=nil {
+          log.Printf("failed to Unmarshaling:%s",err)
+          return c.String(http.StatusInternalServerError,"")
+  }
+  client, err := mongo.NewClient(options.Client().ApplyURI("mongodb://localhost:27017"))
```

Comment:
Please separate the database connection creation phase from the handler create a stand-alone function for it.